src/features/student/views/StudentProjectSkillsView/components/AddAdditionalSkillDrawer/use-additional-skill-form/use-additional-skill-form.ts

```diff
@@ -1,9 +1,22 @@
+import type { BaseApiException } from '@/common/exceptions'
 import type { AdditionalSkillFormData } from 'src/features/student/views/StudentProjectSkillsView/components/AddAdditionalSkillDrawer/types'
+import { useCreateAdditionalSkillMutation } from '@/features/student/queries'
+import { useToasterStore } from '@/store'
 import { useForm } from '@tanstack/vue-form'
 import { useI18n } from 'vue-i18n'
 
 export function useAdditionalSkillForm (onSkillAdded?: () => void) {
   const { t } = useI18n()
+  const { addErrorMessage } = useToasterStore()
+
+  const onCreateAdditionalSkillError = (error: BaseApiException) => {
+    addErrorMessage({
+      title: t('student.views.studentProjectSkillsView.skillsViewTabs.skillsViewOtherTab.addAdditionalSkillDrawer.errors.createAdditionalSkill'),
+      description: error.message
+    })
+  }
+
+  const createAdditionalSkillMutation = useCreateAdditionalSkillMutation({ onError: onCreateAdditionalSkillError })
```

Comment:
```suggestion
  const {mutate: createAdditionalSkill, isPending} = useCreateAdditionalSkillMutation({ onError: onCreateAdditionalSkillError })
```
Pourquoi ne pas faire ça directement ?